bridge/client/app/app-header/app-header.component.html

```diff
@@ -19,6 +19,12 @@
 </dt-top-bar-navigation>
 <dt-context-dialog #dialog aria-label="User menu" ariaLabelClose="Close user menu">
   <h3 class="m-0">Get started</h3>
+  <div class="mt-1" fxLayout="row" fxLayoutAlign="flex-start center" *ngIf="keptnInfo?.bridgeInfo?.cliDownloadLink">
+    <div fxFlex>
+      <p class="m-0 mr-2 nobreak">
+        <a [attr.href]="keptnInfo.bridgeInfo.cliDownloadLink" rel="noopener noreferrer" target="_blank">Download keptn CLI</a></p>
```

Comment:
Can you please change it to a capital K --> _Download Keptn CLI_ 

FYI: We stopped differentiating between "keptn CLI" and "Keptn". In the docs it is always Keptn CLI (when not used as command blocks.)

Reply:
Done